### PR TITLE
images: Install cockpit-dashboard for RHEL / Centos

### PR DESCRIPTION
**Pull request by mvollmer**
So that we can use SSH while testing against released versions.

 * [x] image-refresh centos-7
 * [x] image-refresh rhel-7-7
 * [ ] FAIL: image-refresh rhel-7-8

### CODE REVIEW

**Comment by martinpitt** [comment]
Ack. Now you need to request rebuilds for all rhel-7* and centos-7 images, and also remove it again in centos-7.install and friends.

**Comment by cockpituous** [comment]
image-refresh in progress on 3-ci-srv-05.
Log: https://logs.cockpit-project.org/logs/image-refresh-631-20200311-094444/

File: images/scripts/rhel-8-1-distropkg.install [hunk 1]
```diff
@@ -2,4 +2,7 @@
 
 set -e
 
+# remove cockpit distro packages, testing with upstream master
+rpm --erase --verbose cockpit-dashboard
```
Comment (martinpitt) [inline images/scripts/rhel-8-1-distropkg.install]: This shouldn't be necessary. You changed rhel.setup to only install cockpit-dashboard on rhel-7* and centos-7, not on the RHEL images. Dito for the other rhel-8-* scripts.

**Comment by cockpituous** [comment]
image-refresh centos-7 done: https://github.com/cockpituous/bots/commits/image-refresh-centos-7-20200311-102020

**Comment by cockpituous** [comment]
image-refresh in progress on 3-ci-srv-05.
Log: https://logs.cockpit-project.org/logs/image-refresh-631-20200311-102020/

**Comment by cockpituous** [comment]
image-refresh rhel-7-7 done: https://github.com/cockpituous/bots/commits/image-refresh-rhel-7-7-20200311-104747

**Comment by cockpituous** [comment]
image-refresh in progress on 3-ci-srv-05.
Log: https://logs.cockpit-project.org/logs/image-refresh-631-20200311-104848/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-631-20200311-104848/

**Comment by martinpitt** [comment]
docker failed, retrying

**Comment by cockpituous** [comment]
image-refresh in progress on 4-ci-srv-05.
Log: https://logs.cockpit-project.org/logs/image-refresh-631-20200311-115454/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-631-20200311-115454/

**Comment by martinpitt** [comment]
Meh, seems docker is kaputt in RHEL 7.8 nightlies. That's not a good sign that close to the release..

**Comment by martinpitt** [comment]
Docker failure is https://bugzilla.redhat.com/show_bug.cgi?id=1810052. This should get fixed quickly, so let's ignore that image rebuild for now and just update the two others.

**Comment by mvollmer** [comment]
rhel-7-8 is broken because of https://bugzilla.redhat.com/show_bug.cgi?id=1812505

This seems to be very fresh and I expect it to be fixed quickly. But we decided to go ahead without rhel-7-8 here.

**Pull request merged**